packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -734,25 +738,41 @@ export class AppManagementClient implements DeveloperPlatformClient {
     throw new BugError('Not implemented: appPreviewMode')
   }
 
-  async sendSampleWebhook(_input: SendSampleWebhookVariables): Promise<SendSampleWebhookSchema> {
-    outputDebug('⚠️ sendSampleWebhook is not implemented')
+  async sendSampleWebhook(input: SendSampleWebhookVariables): Promise<SendSampleWebhookSchema> {
+    const query = CliTesting
+    const variables = {
+      address: input.address,
+      apiKey: input.api_key,
+      apiVersion: input.api_version,
+      deliveryMethod: input.delivery_method,
+      sharedSecret: input.shared_secret,
+      topic: input.topic,
+    }
+    const result = await webhooksRequest(query, await this.token(), variables)
+
     return {
       sendSampleWebhook: {
-        samplePayload: '',
-        headers: '{}',
-        success: true,
-        userErrors: [],
+        samplePayload: result.cliTesting?.samplePayload ?? '{}',
```

Comment:
Lots of `?.` here. Should `cliTesting` always be present, is it an error if its not? Maybe the schema should be adjusted?